ims/cli/cli.py

```diff
@@ -671,5 +673,47 @@ def download():
     click.echo('Not Yet Implemented')
 
 
+@cli.group(short_help='Image Related Commands')
+def image():
+    """
+    Use The Subcommands under this command to manipulate Snapshots
```

Comment:
Since we have moved toward the direction where this is not going to be in core BMI in the long run, but instead would more likely be a separate service, I renamed the group to just be "script" so it will be easier to break it out of the codebase.